packages/electric/lib/tasks/styles.js

```diff
@@ -9,10 +9,11 @@ module.exports = function(options) {
 	const pathSrc = options.pathSrc;
 	const sassOptions = options.sassOptions;
 	const taskPrefix = options.taskPrefix;
+	const scssSrc = `${options.scssSrc}/**/*.scss`;
 
 	gulp.task(taskPrefix + 'styles', function() {
 		return gulp
-			.src(path.join(pathSrc, 'styles/**/*.scss'))
+			.src(path.join(pathSrc, scssSrc))
 			.pipe(sass(sassOptions).on('error', sass.logError))
 			.pipe(gulp.dest(path.join(pathDest, 'styles')));
```

Comment:
The output will continue to go to `styles` folder, is that ok for you @clovisdasilvaneto?

Reply:
I think is a bit bad @zenorocha because in the Liferay portal the output folder is named 'css'. In this case I'm setting the default folder to be dynamic, being chosen by the user. 

In Global Services we want to use electric as a theme boilerplate as initial styleguide for our GS themes, it will bring more value for the deliveries of the designs. I've already managed to integrate the electric with Liferay theme, but I've noticed that I have to put my SCSS files on 'styles' folder, but I'd like to change it to 'css' folder, because it is the default name of the Liferay theme, is that possible?

Reply:
Hi, Zeno :).

That's the idea like Clovis said.

Maybe it would be great if we could use Electric as a StyleGuide Doc to our customer's theme when we deliver a project. Currently, this has been one of the deliverables in GS projects.

With a well-designed style guide, the customer easily could follow the UI patterns defined in the theme in order to continue with the good practices and UI patterns. The point using Electric is: Electric needs to "see" the theme .scss or main.css compiled file, and this was the motivation of @clovisdasilvaneto when opened this PR.

Reply:
Hey @clovisdasilvaneto, @jordanamorais, not having more context, and while I'm not against this PR at all, it feels a bit weird.

We have specific tooling to work with Liferay Portal Themes. Electric es meant to create static sites, not Liferay Themes. This feels like opening the door for having to support a lot of the quirks of developing and deploying Themes into Liferay Portal, which I'm pretty sure we don't want to do.

@clovisdasilvaneto, @jordanamorais, could you share a bit more about this? I honestly think GS should be using the official theme tooling and ask for improvements where the existing features are not matching your expectations.

Reply:
Hey @brunobasto, could you reach out to see if we can find a solution for this? Otherwise, we can just close the PR and revisit if more information is found to continue the conversation.

Reply:
Hey @jbalsas, they understand electric is not a tool for developing themes. They are not using it with that purpose. But they would find it useful to have it grab existing css from a theme folder. I talked to Clovis a while ago, and he was ok with finding another way to achieve what he wants if we do not want to add this configuration in.

Reply:
I don't see a problem with this specific configuration, but we they should be aware that we do not support their use case as this was not what electric was designed for. If further changes would be required for his use case, it would probably be out of the scope of this project.

Reply:
Sounds good to me, thanks for clarifying! I'll defer to @Robert-Frampton and @zenorocha regarding mergeability 😉

Reply:
Merged to upstream.